PowerShell/script.ps1

```diff
@@ -0,0 +1,89 @@
+<#===========================================================================================================================================================
```

Comment:
remove this, please. This is example code and should not distract unnecessary  ascii art